src/problem-2.js

```diff
@@ -0,0 +1,16 @@
+function sum(list) {
+    let answer = 0;
+    for (let i = 0; i < list.length; i++) {
+        answer = answer + list[i];
+    }
+    return answer;
+};
+
+const sumES6 = (list) => {
+    return list.reduce((a, b) => a + b)
```

Comment:
A good practice would be to produce a second argument to reduce -- the initial value. It will be automatically inferred from the arguments otherwise.